src/Json.Schema.ToDotNet.UnitTests/Json.Schema.ToDotNet.UnitTests.csproj

```diff
@@ -13,7 +13,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <ItemGroup>
-    <None Include="TestData\*.schema.json">
```

Comment:
>schema [](start = 30, length = 6)

The test data directory used to contain just a few schema files at its root. Now it contains a hierarchy of files with various file name extensions. We have to copy all of them.